scripts/update-monitoring-mixin-dashboards.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+### This script is used to copy dashboards
+## from Monitoring Mixins website repo.
+## https://monitoring.mixins.dev/
+## the repo behind this website is located:
+## https://github.com/monitoring-mixins/website
+## We copy certain dashboards from this repo:
+## alertmanager
```

Comment:
The docs in the script mentions the upstream.
and which dashboards we are copying.